locale/en/blog/release/v6.0.0.md

```diff
@@ -0,0 +1,171 @@
+
+In accordance with our [Long Term Support plan](https://github.com/nodejs/LTS/), Node.js v4 'Argon' will continue to be supported as an *Active* Long Term Support stream until April of 2017, after which it will transition into what we call *Maintenance* LTS. The new Node.js v6 branch will become the new *Active* Long Term Support stream in October of this year, and will continue to be supported actively until April of 2018. If our LTS plan is new to you, [read more](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd#.z6bzdfogi) about how it works so that you can make a properly informed decision regarding which version of Node.js to adopt.
+
+The general rule for deciding which version of Node.js to use is:
+
+* Stay on or upgrade to Node.js v4 (currently at v4.4.3) if you need stability and have a complex production environment.
+* Upgrade to Node.js v6 if you have the ability to upgrade versions quickly and want to play with the latest features as they arrive.
+
+The release notes below are annotated with the main breaking changes that warrant the jump to v6. Note that because this new version of Node.js is shipping with a new version of V8, you will need to recompile any native addons you have already installed or you will receive a runtime error when trying to load them. Use `npm rebuild` or simply remove your *node_modules* directory and `npm install` from scratch.
+
+### What about Node.js v5?
+
+Node.js v5 will continue to be supported for the next two months in order to give developers currently using v5 time to transition to Node.js v6.
+
+### What about Node.js v0.10 and v0.12?
+
+If you're still currently using Node.js v0.10 or v0.12, it is time to begin the transition to v4 or v6. Both of these versions are considered to be in *Maintenance* mode currently and will fall off our support plan completely later this year.
+
+### Current? What happened to Stable?
+
+You may have noticed that the v6 release uses the label `Current` instead of `Stable`. This name change has been made in order to better differentiate the Long Term Support releases from the current active development branches. We
+will be using the `Current` label from here-on instead of `Stable`.
+
+### Notable Changes
+
+The following significant changes have been made since the previous Node.js v5.0.0 release.
+
+* Buffer
+  * New Buffer constructors have been added
+    [#4682](https://github.com/nodejs/node/pull/4682) and
+    [#5833](https://github.com/nodejs/node/pull/5833).
+  * Existing `Buffer()` and `SlowBuffer()` constructors have been deprecated
+    in docs [#4682](https://github.com/nodejs/node/pull/4682) and
```

Comment:
fair enough... specifically mentioned it as it was inconsistent with other changes.

Total nit though :smile: